packages/rest-crud/README.md

```diff
@@ -29,18 +28,79 @@ npm install --save @loopback/rest-crud
    >(Product, {basePath: '/products'});
    ```
 
-4. Set up dependency injection for the ProductController.
+2. Set up dependency injection for the ProductController.
 
    ```ts
    inject('repositories.ProductRepository')(ProductController, undefined, 0);
    ```
 
-5. Register the controller with your application.
+3. Register the controller with your application.
 
    ```ts
    app.controller(ProductController);
    ```
 
+## Creating a CRUD repository
+
+Use the `defineCrudRepositoryClass` method to create named repositories (based
+on the Model) for your app.
+
+Usage example:
+
+```ts
+const db = new juggler.DataSource({connector: 'memory'});
+const ProductRepository = defineCrudRepositoryClass(Product);
+const repo = new ProductRepository(db);
+```
+
+## Integrated example
+
+Here is an example of an app which uses `defineCrudRepositoryClass` and
+`defineCrudRestController` to fulfill its repository and controller
+requirements. You don't need to run `lb4 model` and `lb4 repository` to generate
+the model and respository classes.
```

Comment:
Interesting. In my mind, I wanted the example to keep as much of existing CLI & Boot setup as possible, i.e. use `lb4 model` and `lb4 datasource`, and write custom code for the repository & controller only. That's why I proposed to remove creation of the datasource from the boot method and assume it will be created by `@loopback/boot` from `src/datasources/db.datasource.{js,json}`.

If you prefer to create the model & datasource via code, then I am fine with that too, but we need to find out how to deal with `@loopback/boot`. Do we want to mix artifacts loaded by boot with artifacts defined from code? Do we want those two kinds of artifacts to share the same datasource?